angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.component.spec.ts

```diff
@@ -31,17 +31,52 @@ describe('SprkIconComponent', () => {
     expect(component.getClasses()).toEqual('sprk-c-Icon');
   });
 
+  it('should add the correct icon based on the icon type set', () => {
```

Comment:
Ope, this should be icon name set since we updated the Input

Reply:
✅ fixed this and pushed up